subworkflows/local/locidex.nf

```diff
@@ -22,11 +21,15 @@ workflow LOCIDEX {
             meta, top_hit, contigs -> tuple(meta, contigs, file(params.allele_scheme))
         }
 
+        // TODO add test for report value here
```

Comment:
TODO

Reply:
Actually doing this TODO now...

Reply:
After reflecting on this PMs will likely want the metadata output to be the same for databases. So I am updating this logic to include this change.

Reply:
TODO is gone: https://github.com/phac-nml/mikrokondo/pull/96/commits/673349a66cf33c4303f62af055f7e8b1bb87337a